src/main/java/org/opentripplanner/gtfs/mapping/GTFSToOtpTransitServiceMapper.java

```diff
@@ -95,6 +111,13 @@ public OtpTransitServiceBuilder getBuilder() {
     }
 
     public void mapStopTripAndRouteDatantoBuilder() {
+        String feedLanguage = null;
+        if (data.getAllFeedInfos().iterator().hasNext()) {
+            feedLanguage = data.getAllFeedInfos().iterator().next().getLang();
+        }
+
+        TranslationHelper translationHelper =
+                new TranslationHelper(data.getAllTranslations(), feedLanguage);
```

Comment:
Move to constructor?

Reply:
Moved to constructor.

Reply:
It is still in the `mapStopTripAndRouteDatantoBuilder`, not the constructor?